x-pack/plugins/apm/server/lib/service_map/get_service_map_service_node_info.ts

```diff
@@ -71,32 +73,24 @@ export async function getServiceMapServiceNodeInfo({
   };
 }
 
-async function getErrorMetrics({ setup, minutes, filter }: TaskParameters) {
-  const { client, indices } = setup;
-
-  const response = await client.search({
-    index: indices['apm_oss.errorIndices'],
-    body: {
-      size: 0,
-      query: {
-        bool: {
-          filter: filter.concat({
-            term: {
-              [PROCESSOR_EVENT]: 'error',
-            },
-          }),
-        },
-      },
-      track_total_hits: true,
-    },
+async function getErrorMetrics({
+  environment,
+  serviceName,
+  setup,
+}: {
+  environment?: string;
+  serviceName: string;
+  setup: Options['setup'];
+}) {
+  const groupId = undefined;
+  const { average, noHits } = await getErrorRate({
+    serviceName,
+    groupId,
+    environment,
+    setup: { ...setup, uiFiltersES: [] },
   });
 
-  return {
-    avgErrorsPerMinute:
-      response.hits.total.value > 0
-        ? response.hits.total.value / minutes
-        : null,
-  };
+  return { avgErrorRate: noHits ? null : average };
 }
 
 async function getTransactionMetrics({
```

Comment:
Can we find another term than "metrics"? It's overloaded with different meaning: metric based ui; agent metrics; metric index - we even have a metrics team :) 

I don't have any great suggestions but even something like `getTransactionStats` will be an improvement